builtin/logical/pki/path_revoke.go

```diff
@@ -71,6 +84,28 @@ signed by an issuer in this mount.`,
 				// If this needs to write, the entire request will be forwarded to the
 				// active node of the current performance cluster, but we don't want to
 				// forward invalid revoke requests there.
+				Responses: map[int][]framework.Response{
+					http.StatusOK: {{
+						Description: "OK",
+						Fields: map[string]*framework.FieldSchema{
+							"revocation_time": {
+								Type:        framework.TypeDurationSecond,
+								Description: `Revocation Time`,
+								Required:    false,
+							},
+							"revocation_time_rfc3339": {
+								Type:        framework.TypeTime,
+								Description: `Revocation Time`,
+								Required:    false,
+							},
+							"state": {
+								Type:        framework.TypeString,
+								Description: `Revocation State`,
+								Required:    false,
+							},
```

Comment:
I did not see this one in the code, could you please double-check.

Reply:
It looks like this can get added [here](https://github.com/hashicorp/vault/blob/8651d6de38890ae5d74903d21971386b915b81e5/builtin/logical/pki/path_revoke.go#LL470C23-L470C23) 

and adds that `state`

Reply:
it also goes through this path and can get added [here](https://github.com/hashicorp/vault/blob/8651d6de38890ae5d74903d21971386b915b81e5/builtin/logical/pki/crl_util.go#L1023)

Reply:
@AnPucel is correct :-)

Reply:
Ah completely missed that code path, thanks for checking!